### PR TITLE
Mp/main responsive

### PR DESCRIPTION
**Pull request by milos381**
### Summary

_Provide a short summary of the changes in this PR_

### Type of Change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

### Issues

-   Link any issues this PR resolves using keywords (resolve, closes, fixed)

### Evidence

-   Provide evidence of the the changes functioning as expected or describe your tests. If tests are included in the CI pipeline this may be omitted.

_(delete this line)_ Prior to submitting the PR assign a reviewer from each team to review this PR.

### CODE REVIEW

**Review by feiw101 — COMMENTED** [review]
Awesome!! No request to changes on my end. Will approve once all the previous PRs got merged into dev.

**Review by evanmcneely — APPROVED** [review]

**Comment by shalarewicz** [comment]
Changes included in #88

**Pull request closed without merging**
